WordPress/Classes/ViewRelated/Post/FeaturedImageViewController.m

```diff
@@ -27,7 +27,13 @@ - (void)viewDidLoad
     [super viewDidLoad];
 
     self.title = NSLocalizedString(@"Featured Image", @"Title for the Featured Image view");
-    self.view.backgroundColor = [UIColor whiteColor];
+    
+    if (@available(iOS 13, *)) {
+        self.view.backgroundColor = [UIColor systemBackgroundColor];
```

Comment:
Rather than doing the iOS 13 check here, you can just set the background color to `[UIColor basicBackground]`. We define it ourselves in UIColor+MurielColors, and it does this same switch - white in iOS 12, system background in iOS 13 :)

Reply:
Fixed this one with an update to using the muriel colors too.